src/mixins/Themeable.ts

```diff
@@ -26,10 +26,10 @@ export type ClassNames = {
 /**
  * Properties required for the themeable mixin
  */
-export interface ThemeableProperties extends WidgetProperties {
+export interface ThemeableProperties<T = any> extends WidgetProperties {
```

Comment:
Is `any` the true default of this, nothing more narrow, even a simple `{ [key: string]: any; }`?